include/xrpl/protocol/STIssue.h

```diff
@@ -31,31 +31,40 @@ namespace ripple {
 class STIssue final : public STBase, CountedObject<STIssue>
 {
 private:
-    Issue issue_{xrpIssue()};
+    Asset asset_{xrpIssue()};
```

Comment:
Is name `xrpIssue()` still accurate ?

Reply:
Yes. It's a special `Issue` of `currency{0}/account{0}`.

Reply:
[](https://docs.google.com/spreadsheets/d/11YB9WC8kXtH34an-LPdC3KF5GEqP70RANAdkwbW55rY/edit)